mantle/cmd/kola/qemuexec.go

```diff
@@ -96,11 +101,35 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 		}
 		config = *newconfig
 	}
+	for _, b := range bindro {
+		s := strings.Split(b, ",")
+		src := s[0]
+		dest := s[1]
+		config = v3.Merge(config, conf.Mount9p(src, dest, true))
+	}
+	for _, b := range bindrw {
+		s := strings.Split(b, ",")
+		src := s[0]
+		dest := s[1]
+		config = v3.Merge(config, conf.Mount9p(src, dest, false))
+	}
 	builder, err := platform.NewBuilderFromParsed(config, kola.Options.IgnitionVersion == "v2")
 	if err != nil {
 		return errors.Wrapf(err, "creating qemu builder")
 	}
 	builder.ForceConfigInjection = forceConfigInjection
+	for _, b := range bindro {
+		s := strings.Split(b, ",")
+		src := s[0]
+		dest := s[1]
+		builder.Mount9p(src, dest, true)
+	}
+	for _, b := range bindrw {
+		s := strings.Split(b, ",")
+		src := s[0]
+		dest := s[1]
+		builder.Mount9p(src, dest, false)
+	}
```

Comment:
Hmm, the code duplication here is a bit unfortunate. Couldn't we just do both the config and builder mounts at the same time here and just use `builder.Config`?

Reply:
This is https://github.com/coreos/coreos-assembler/pull/1285#discussion_r398031225 right?

To elaborate a bit, your suggestion mutates the config we passed to the builder via `platform.NewBuilderFromParsed`, and in fact right now (to avoid more refactoring) that API call immediately converts the Ignition config to a string so it wouldn't work.

A few ways we could break this dependency:

- Support setting an Ignition config *after* builder is created
- Make `builder.Mount9p` also mutate the Ignition config

(or both)

Not opposed to either of these but OTOH it feels like it can also be a cleanup afterwards?  Would reduce the conflicts with already dependent PRs.

Reply:
Ehh I will do at least the first one now.

Reply:
https://github.com/coreos/coreos-assembler/pull/1291

Reply:
And now rebased on that and did further cleanups.